src/config.rs

```diff
@@ -2791,10 +2791,16 @@ mod tests {
         let mut last_cfg_metadata = last_cfg_path.metadata().unwrap();
         let first_modified = last_cfg_metadata.modified().unwrap();
 
+        // not write to file when config is the equivalent of last one.
         assert!(persist_config(&tikv_cfg).is_ok());
-
         last_cfg_metadata = last_cfg_path.metadata().unwrap();
         assert_eq!(last_cfg_metadata.modified().unwrap(), first_modified);
```

Comment:
```
test config::tests::test_last_cfg_modified ... thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `SystemTime { tv_sec: 1598249476, tv_nsec: 91041229 }`,
 right: `SystemTime { tv_sec: 1598249476, tv_nsec: 82041320 }`', src/config.rs:2797:9
```
Why are two `modified` time are same in seconds but different in nanoseconds?

Reply:
Then `persist_config` is triggering writes unexpectedly.

Reply:
I've fixed it.